concepts/Enarx_Public_Chat_App_Zulip/000005-Enarx_Public_Chat_App_Zulip.md

```diff
@@ -0,0 +1,126 @@
+| Dark mode                                   | Yes                                                                                                                                                           | Yes                                                                                                                                                                                   |
+| Emoji reactions to messages                 | No                                                                                                                                                            | Yes                                                                                                                                                                                   |
+| Custom emoji                                | No                                                                                                                                                            | Yes                                                                                                                                                                                   |
+
+## Drawbacks
+
+There are two major drawbacks to Zulip we should strongly consider before a move.
+
+**Community**
+
+The community we’ve begun building up on Gitter is easily the biggest reason we have to stay. If we move chat clients, we’d be asking them to juggle around yet another chat app, and their existing lines of communication would end up unsupported by us. This is not ideal, and a big con to moving.
+
+The counter-argument to this is that if a change is going to happen, it’s better that it happens sooner rather than later. If we truly feel that Gitter won’t meet our needs (especially as we scale), a change now will be far less painful than when we’re a lot larger.
+
+**Public chat visibility**
+
+The biggest thing Gitter does significantly _better_ than Zulip is the ability to read messages when logged out. If someone new to the project is idly curious what's going on in chat, they can just look at our chat log in Gitter -- no login required.
+
+Zulip, on the other hand, requires a login, which presents a barrier to people being able to see the discussions going on about Enarx. An idle observer would need to create an account, which they might not want to do.
+
+There is a potential workaround present, however, with [`zulip-archive`](https://github.com/zulip/zulip-archive). This is a tool, runnable as a Github Action, that archives our chat logs and publishes them on a publicly-accessible Github Pages site. For Enarx, this may manifest itself as a sub-link of `enarx.github.io` -- for example, we may be able to point people to `enarx.github.io/zulip-archive` for a publicly-accessible chat record.
```

Comment:
Two small things.

I think we should avoid using the enarx.github.io domain and stick to enarx.io, which is better in terms of "branding".

We could also make the name of the chat more generic, for future proofing. So `chat.enarx.io` as a canonical chat URL and maybe `chat.enarx.io/archive`.

```suggestion
There is a potential workaround present, however, with [`zulip-archive`](https://github.com/zulip/zulip-archive). This is a tool, runnable as a Github Action, that archives our chat logs and publishes them on a publicly-accessible Github Pages site. For Enarx, this may manifest itself as a sub-link of `enarx.io` -- for example, we may be able to point people to `chat.enarx..io/archive` for a publicly-accessible chat record.
```